react/Autosuggest/Autosuggest.js

```diff
@@ -5,10 +5,13 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-import omit from 'lodash.omit';
+import { get, invoke, omit } from 'lodash';
```

Comment:
This is going to result in lodash being included in its entirety for some users, we should import the things we need individually.